src/documentation/app/layout/content/components/LinkItem.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
 
-const LinkWrapper = styled(Link)`
+const linkWrapperStyleString = `
```

Comment:
I imagine there's a better way to reuse the same styles for both the `<a>` and the `<Link>` instead of this variable that gets interpolated.